editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -29,7 +29,7 @@ See also:
 
 If the button doesn't work save this link:
 
-<a href="http://tiddlywiki.com/index.html" download="index.html">~http://tiddlywiki.com/index.html</a>
+<a href="http://tiddlywiki.com/index.html" download="" rel="noreferrer">~http://tiddlywiki.com/index.html</a>
```

Comment:
Did you intend to clear the `download` attribute? It's probably better to drop the `=""` as it's now redundant.